pkgs/tools/misc/immich-cli/default.nix

```
@@ -1,27 +1,29 @@
-{ lib
-, buildNpmPackage
-, fetchFromGitHub
-}:
-
+{ lib, buildNpmPackage, fetchFromGitHub }:
```

Let's not make unnecessary changes.
```suggestion
{ lib
, buildNpmPackage
, fetchFromGitHub
}:

```